tests/PyPI-pytest/run_all.py

```diff
@@ -289,11 +129,9 @@ def main():
         if package_name in (
             "google-auth",  # bdist_nuitka fails AttributeError: single_version_externally_managed
             "jinja2",  # ModuleNotFoundError: No module named 'jinja2.tests'
-            "pandas",  # python setup.py egg_info fails
-            "pytz",  # AssertionError: zoneinfo files not found!
-            "setuptools",  # compiled __import__ check fails; pytest passes
-            "wheel",  # compiled __import__ check fails; pytest passes
-            "pytest",  # compiled __import__ check fails; should we even test this?
+            "pandas",  # ModuleNotFoundError: No module named 'Cython'
+            "pytz",  # need to 'make build'
+            "rsa",  # Now uses Poetry (no setup.py)
```

Comment:
This poetry thing is something I encountered elsewhere too, can you look into how it could be plugged with Nuitka?

Reply:
they are using a `tox.ini` file instead of `setup.py` from what I see

Reply:
I think `tox.ini` is just a way of making it test with multiple Python versions, but in the end, it should be still something like `py.test`, esp. with tox I think it's extremely common. What is in that tox.ini ?